src/olympia/editors/helpers.py

```diff
@@ -670,6 +729,8 @@ def request_information(self):
         self.log_action(amo.LOG.REQUEST_INFORMATION)
         if self.version:
             self.version.update(has_info_request=True)
+            if not self.addon.is_listed and not self.version.reviewed:
+                self.version.update(reviewed=datetime.datetime.now())
```

Comment:
We need to set `reviewed` for Unlisted add-ons because the comment will be the 'review' the reviewer does, even though the status won't change (`reviewed` is otherwise set in the approval, etc, process methods for Listed add-ons that aren't auto-approved)